lib/resources/r4/immunization.yaml

```diff
@@ -0,0 +1,83 @@
+---
+name: Immunization
+field_name_base_url: http://hl7.org/fhir/R4/immunization-definitions.html#Immunization
+fields:
+
+- name: statusReason
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Reason the immunization was not performed.
+  binding:
+    description: Reason the immunization was not performed.
+    terminology:
+    - display: Immunization Status Reason
+      system: http://terminology.hl7.org/CodeSystem/v3-ActReason
+      info_link: http://hl7.org/fhir/r4/valueset-immunization-status-reason.html
+    - display: Millennium Expire Reasons
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/30440
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-30440-expire-reasons
+
+- name: vaccineCode
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: The vaccine that was administered or was to be administered.
+  binding:
+    description: Vaccine product administered.
+    terminology:
+    - display: CVX
+      system: http://hl7.org/fhir/sid/cvx
+      info_link: http://hl7.org/fhir/r4/cvx.html
+    - display: NDC
+      system: http://hl7.org/fhir/sid/ndc
+      info_link: http://hl7.org/fhir/r4/ndc.html
+    - display: Millennium Clinical Event Code
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/72
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-72-clinical-event-codes
+    note: The CVX binding will always be returned. An NDC binding will be returned when available.
+
+- name: reportOrigin
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: The source of an immunization record when not reported by the person who administered the vaccine.
+  binding:
+    description: The source of an immunization record when not reported by the person who administered the vaccine.
+    terminology:
+    - display: Immunization Report Origin
+      system: http://hl7.org/fhir/R4/codesystem-immunization-origin.html
```

Comment:
Pretty sure it should be this: http://terminology.hl7.org/CodeSystem/immunization-origin

Reply:
Updated. https://github.com/cerner/fhir.cerner.com/pull/255/commits/3fa313c0f09253444d22d4ea494e61e108042e75#diff-2c50e8f41daaba3a640861e7b6e81488L49

Reply:
Is it not returning the terminology link? it should

Reply:
Yes. Both of the following terminology links are not returning anything.
https://terminology.hl7.org/CodeSystem/immunization-origin
https://terminology.hl7.org/CodeSystem/v3-ActReason

Reply:
@Likitha-Seeram change the 'system' to 'http://terminology.hl7.org/CodeSystem/immunization-origin'

Reply:
Updated. https://github.com/cerner/fhir.cerner.com/pull/255/commits/8ef53a1253acad23197ba1804453f7548b71bad5#diff-2c50e8f41daaba3a640861e7b6e81488L49

Reply:
+1